pre_build.py

```diff
@@ -14,8 +14,15 @@ def build() -> None:
         )
         exit(1)
     print("Building front end...")
-    subprocess.run(["yarn", "webui:install"])
-    subprocess.run(["yarn", "webui:build"])
+    try:
+        if os.name == "nt":
+            subprocess.check_output(" ".join(["yarn", "webui:install"]), shell=True)
+            subprocess.check_output(" ".join(["yarn", "webui:build"]), shell=True)
+        else:
+            subprocess.run(["yarn", "webui:install"])
```

Comment:
Doesn't 
`subprocess.check_output("yarn webui:install", shell=True)` work on both platforms? I'd prefer that, to avoid the if-statement.

Reply:
It may do! Let me check

Reply:
Looks sound